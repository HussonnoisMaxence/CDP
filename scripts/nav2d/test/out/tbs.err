Traceback (most recent call last):
  File "./Projects/CDP/plot_from_tb.py", line 269, in <module>
    main(args)
  File "./Projects/CDP/plot_from_tb.py", line 259, in main
    plot_speed(path)
  File "./Projects/CDP/plot_from_tb.py", line 208, in plot_speed
    timesteps = dic['0.3'][0][dic['0.3'][0].metric.str.endswith("0-speed")].step.to_numpy()
  File "/home/s222126776/Projects/CDP2/env/lib/python3.8/site-packages/pandas/core/generic.py", line 5575, in __getattr__
    return object.__getattribute__(self, name)
  File "/home/s222126776/Projects/CDP2/env/lib/python3.8/site-packages/pandas/core/accessor.py", line 182, in __get__
    accessor_obj = self._accessor(obj)
  File "/home/s222126776/Projects/CDP2/env/lib/python3.8/site-packages/pandas/core/strings/accessor.py", line 177, in __init__
    self._inferred_dtype = self._validate(data)
  File "/home/s222126776/Projects/CDP2/env/lib/python3.8/site-packages/pandas/core/strings/accessor.py", line 231, in _validate
    raise AttributeError("Can only use .str accessor with string values!")
AttributeError: Can only use .str accessor with string values!
