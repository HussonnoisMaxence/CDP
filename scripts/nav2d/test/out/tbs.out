['/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/89067/0.3/run_back/tb/events.out.tfevents.1666853854.4gpu-10.3014322.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/45123/0.3/run_back/tb/events.out.tfevents.1666853853.4gpu-10.3014298.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/12345/0.3/run_back/tb/events.out.tfevents.1666853854.4gpu-10.3014292.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/78906/0.3/run_back/tb/events.out.tfevents.1666853852.4gpu-10.3014358.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/6789/0.3/run_back/tb/events.out.tfevents.1666853854.4gpu-10.3014312.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/23451/0.3/run_back/tb/events.out.tfevents.1666853851.4gpu-10.3014328.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/90678/0.3/run_back/tb/events.out.tfevents.1666853848.4gpu-10.3014326.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/34512/0.3/run_back/tb/events.out.tfevents.1666853852.4gpu-10.3014297.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/51234/0.3/run_back/tb/events.out.tfevents.1666853855.4gpu-10.3014324.0results-0.3', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/67890/0.3/run_back/tb/events.out.tfevents.1666853855.4gpu-10.3014363.0results-0.3']
['/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/89067/0.5/run_back/tb/events.out.tfevents.1666853855.4gpu-10.3014580.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/45123/0.5/run_back/tb/events.out.tfevents.1666853849.4gpu-10.3014558.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/12345/0.5/run_back/tb/events.out.tfevents.1666853848.4gpu-10.3014546.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/78906/0.5/run_back/tb/events.out.tfevents.1666853852.4gpu-10.3014588.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/6789/0.5/run_back/tb/events.out.tfevents.1666853854.4gpu-10.3014586.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/23451/0.5/run_back/tb/events.out.tfevents.1666853853.4gpu-10.3014563.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/90678/0.5/run_back/tb/events.out.tfevents.1666853848.4gpu-10.3014581.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/34512/0.5/run_back/tb/events.out.tfevents.1666853850.4gpu-10.3014548.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/51234/0.5/run_back/tb/events.out.tfevents.1666853853.4gpu-10.3014590.0results-0.5', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/67890/0.5/run_back/tb/events.out.tfevents.1666853850.4gpu-10.3014556.0results-0.5']
['/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/89067/0.9/run_back/tb/events.out.tfevents.1666853888.8gpu-5.2047682.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/45123/0.9/run_back/tb/events.out.tfevents.1666853887.8gpu-5.2047663.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/12345/0.9/run_back/tb/events.out.tfevents.1666853889.8gpu-5.2047641.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/78906/0.9/run_back/tb/events.out.tfevents.1666853891.8gpu-5.2047684.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/6789/0.9/run_back/tb/events.out.tfevents.1666853886.8gpu-5.2047680.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/23451/0.9/run_back/tb/events.out.tfevents.1666853890.8gpu-5.2047665.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/90678/0.9/run_back/tb/events.out.tfevents.1666853890.8gpu-5.2047675.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/34512/0.9/run_back/tb/events.out.tfevents.1666853887.8gpu-5.2047639.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/51234/0.9/run_back/tb/events.out.tfevents.1666853888.8gpu-5.2047676.0results-0.9', '/home/s222126776/Projects/CDP/experiments/Mujoco/HC/beta3/67890/0.9/run_back/tb/events.out.tfevents.1666853888.8gpu-5.2047643.0results-0.9']
